Source/Scene/WebMapServiceImageryProvider.js

```diff
@@ -165,6 +196,15 @@ define([
         });
     }
 
+    function requestImage(imageryProvider, col, row, level, request, interval) {
+        var dynamicIntervalData = defined(interval) ? interval.data : undefined;
```

Comment:
Following the Cesium style guidelines we save off any properties accessed more than once in the same function. Here for instance we would do something like the following because `imageryProvider._tileProvider` is used a few times.
```javascript
var tileProvider = imageryProvider._tileProvider;
if (defined(dynamicIntervalData)) {
    tileProvider._resource.setQueryParameters(dynamicIntervalData);
}
return tileProvider.requestImage(col, row, level, request);
```

Also modifying `tileProvider._resource` shouldn't be an issue, since we create a copy of the resource before any async calls are made.